docusaurus.config.js

```diff
@@ -0,0 +1,78 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: 'BotKube :: Messaging bot for monitoring and debugging Kubernetes clusters',
+  tagline: 'BotKube is cool',
+  url: 'https://botkube.io/',
+  baseUrl: '/',
+  onBrokenLinks: 'throw',
+  onBrokenMarkdownLinks: 'warn',
+  favicon: 'images/favicon.png',
+
+  // GitHub pages deployment config.
+  // If you aren't using GitHub pages, you don't need these.
+  organizationName: 'KubeShop', // Usually your GitHub org/user name.
+  projectName: 'botkube', // Usually your repo name.
```

Comment:
We'll use Cloudflare Pages anyway, but we can keep them. Let's remove comments though:

```suggestion
  organizationName: 'kubeshop",
  projectName: 'botkube',
```